core/vm.cpp

```diff
@@ -869,7 +869,12 @@ class Interpreter {
         builtins["extVar"] = &Interpreter::builtinExtVar;
         builtins["primitiveEquals"] = &Interpreter::builtinPrimitiveEquals;
         builtins["native"] = &Interpreter::builtinNative;
+#ifndef DISABLE_INSECURE_HASH
         builtins["md5"] = &Interpreter::builtinMd5;
```

Comment:
Any reason to leave the old one at all? Why not just always use OpenSSL version if we depend on it anyway?

Reply:
Backwards compatibility. We don't want to force people to redeploy their entire infrastructure because they were using md5 to generate unique names for things.